src/main/java/frc/robot/commands/IntakeSequence.java

```diff
@@ -9,26 +9,34 @@
 import frc.robot.subsystems.elevator.Elevator.ElevatorPosition;
 import frc.robot.subsystems.intake.Intake;
 import frc.robot.subsystems.intake.Intake.IntakePosition;
+import frc.robot.subsystems.shooter.Shooter;
+import frc.robot.subsystems.shooter.Shooter.ShooterMode;
 
 /**
  * Puts the intake into the deployed position then gets elevator to home position Checks that
  * everything is in position to intake then starts rollers Intakes until the conveyor recieves the
  * game piece Stops and stowes intake
  */
 public class IntakeSequence extends SequentialCommandGroup {
-  public IntakeSequence(Intake intake, Elevator elevator, Conveyor conveyor) {
-    addRequirements(intake, elevator, conveyor);
+  public IntakeSequence(Intake intake, Elevator elevator, Conveyor conveyor, Shooter shooter) {
+    addRequirements(intake, elevator, conveyor, shooter);
 
     addCommands(
         new InstantCommand(
             () -> {
               intake.setDesiredIntakePosition(IntakePosition.DEPLOYED);
             },
             intake),
+        new InstantCommand(
+            () -> {
+              shooter.setShooterMode(ShooterMode.IDLE);
+            }),
         new ConditionalCommand(
             new InstantCommand(),
-            new WaitUntilCommand(intake::clearOfConveyorZone),
-            elevator::elevatorAboveInterferenceZone),
+            new SequentialCommandGroup(
```

Comment:
This should maybe be parallel? I don't remember exactly which type it is to wait for them both to finish. I suppose functionally it doesn't really matter but it does make it a little clearer IMO

Reply:
I feel this way wherever we have consecutive waits in a sequential command group. It doesn't *really* matter but they shouldn't be thought of as "wait for one then wait for the other" which is what a sequential command group suggests.

Reply:
This is easier to follow and the outcome is the same?

Reply:
I feel like parallel is a little easier to follow but it works the same and that's just my opinion so not a big deal